apps/dashboard/src/lib/features/template/api/template.svelte.ts

```diff
@@ -0,0 +1,39 @@
+import { BaseApi, classroomio } from '$lib/utils/services/api';
+
+import type { GetAllTemplatesMetadataSuccess, GetTemplateByIdSuccess } from '$lib/features/org/utils/types';
+
+/**
+ * API class for exercise templates
+ */
+class TemplateApi extends BaseApi {
```

Comment:
Please rename to `ExerciseTemplateApi` and update other naming that uses only `template`